### PR TITLE
EVG-7928 web form support for TimeLockout 

### PR DESCRIPTION
**Pull request by hhoke**
add web form and backend support for form field so that TimeLockout c…an be set on Admin Page, ref. EVG-7928

### CODE REVIEW

File: service/templates/admin.html [hunk 1]
```diff
@@ -1237,6 +1237,10 @@ <h2 class="modal-title">[[modalTitle]]</h2>
 									<input type="number" ng-model="Settings.amboy.group_ttl">
 									</md-input-container>
 
+				          <md-input-container class="control" style="width:45%;">
+									<label>lock timeout</label>
```
Comment (john-m-liu) [inline service/templates/admin.html:1241]: The label should be in sentence case

Comment (hhoke) [reply service/templates/admin.html:1241]:
```suggestion
									<label>Lock timeout</label>
```
Like that? also thanks for the guidance regarding the Ticket #

Comment (john-m-liu) [reply service/templates/admin.html:1241]: Yep, exactly

File: config_amboy.go [hunk 2]
```diff
@@ -20,6 +20,7 @@ type AmboyConfig struct {
 	GroupPruneFrequencyMinutes            int    `bson:"group_prune_frequency" json:"group_prune_frequency" yaml:"group_prune_frequency"`
 	GroupTTLMinutes                       int    `bson:"group_ttl" json:"group_ttl" yaml:"group_ttl"`
 	RequireRemotePriority                 bool   `bson:"require_remote_priority" json:"require_remote_priority" yaml:"require_remote_priority"`
+	LockTimeout                           int    `bson:"lock_timeout" json:"lock_timeout" yaml:"lock_timeout"`
```
Comment (tychoish) [inline config_amboy.go:23]: can the field name be LockTimeoutMinutes

File: service/templates/admin.html (hunk 1 above)
Comment (tychoish) [reply service/templates/admin.html:1241]: Can we have it read "Lock timeout (minutes)"

**Pull request merged**